Dockerfile

```diff
@@ -45,8 +45,12 @@ RUN set -ex \
   && which docker \
   && (docker version || true)
 
-RUN apt-get install kubectl
-RUN npm -g i npm
-RUN yarn global add nodemon typescript colorguard node-gyp cypress@3.0.3 node-static mocha istanbul bower grunt-cli bower-shrinkwrap-resolver nc
```

Comment:
is it okay for front application to remove `cypress`?

Reply:
Yes they won't use the alpine image, because it is not support on Alpine

Reply:
we'll find another way for the front apps